canvas_modules/common-canvas/src/common-properties/panels/tearsheet/tearsheet.scss

```diff
@@ -16,6 +16,13 @@
 
 $properties-buttons-height: $spacing-10;
 
+.properties-tearsheet-panel.properties-tearsheet-stacked.bx--modal {
+	.bx--modal-container {
+		max-height: calc(100% - 104px);
+		width: calc(100% - 96px);
```

Comment:
Please use spacing token here

Reply:
@nmgokhale Theres no corresponding height in the tokens that will apply for both of these. Im not sure if `96px` is just a coincidence or not for `$spacing-12`. The original Tearsheet heights were [hardcoded as well](https://github.com/elyra-ai/canvas/blob/dbdb80da23908c6e0f8ade1c322432ad0ea70e79/canvas_modules/common-canvas/src/common-properties/panels/tearsheet/tearsheet.scss#L35)